ResearchKit/+Medable/ORKFormItemCell+Medable.m

```diff
@@ -0,0 +1,95 @@
+//
+//  ORKFormItemCell+Medable.m
+//  Axon
+//
+//  Created by me on 5/18/18.
+//  Copyright © 2018 Medable Inc. All rights reserved.
+//
+
+#import "ResearchKit.h"
+#import "ORKFormItemCell.h"
+
+@interface ORKFormItemCell ()
+- (void)ork_setAnswer:(id)answer;
+@end
+
+#pragma mark -
+
+@implementation ORKFormItemTextFieldBasedCell (Medable)
+
+- (void)ork_setAnswer:(id)answer
```

Comment:
Also, would this do the same for every password text entry? We need this to be optional, not all password fields will have a minimum strength and require visual feedback.

Reply:
Or would this be solved by checking `minimumPasswordStrength` below?

Reply:
`minimumPasswordStrength` is checked for a non-zero value. If it's zero, there's no behavioral change for the edit field.